ttnn/cpp/ttnn/operations/data_movement/squeeze/squeeze.cpp

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+
+#include "squeeze.hpp"
+#include "ttnn/operations/core/core.hpp"
+
+namespace ttnn::operations::data_movement {
+
+ttnn::Tensor SqueezeOperation::invoke(
+    const ttnn::Tensor& input_tensor,
+    const int dim
```

Comment:
makes sense to check if shape[dim] == 1, right?

Reply:
in PyTorch if its not 1, it is ignored, I'll add that to the documentation